src/main/java/browser/BrowserImpl.java

```diff
@@ -0,0 +1,31 @@
+package browser;
+
+import org.openqa.selenium.Dimension;
+import webdriver.DriverWrapper;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class BrowserImpl {
```

Comment:
Rename this class to make it clear it's purpose. It's definitely not Browser Implementation